fedWireMessage.go

```diff
@@ -239,9 +246,9 @@ func (fwm *FEDWireMessage) validateTypeSubType() error {
 }
 
 // validateIMAD validates TagInputMessageAccountabilityData within a FEDWireMessage
-// Mandatory for all requests
+// Mandatory for all requests unless sender is a FedLine Advantage customer
 func (fwm *FEDWireMessage) validateIMAD() error {
-	if fwm.InputMessageAccountabilityData == nil {
+	if fwm.InputMessageAccountabilityData == nil && fwm.validateOpts.AllowMissingIMAD == false {
```

Comment:
This is going to panic in most cases because `validateOpts` will be `nil`. I think it'd make sense to update the [`fwm.verify()` method](https://github.com/moov-io/wire/blob/dfe8ae5232990d9a3b0265bb13d8ba2b456c423f/fedWireMessage.go#L138) to do
```go
if fwm.validateOpts == nil {
    fwm.validateOpts = &validateOpts{}
}
```
right at the start.